dockstore-webservice/src/main/java/io/dockstore/webservice/core/Token.java

```diff
@@ -89,31 +89,38 @@
     @SequenceGenerator(name = "token_id_seq", sequenceName = "token_id_seq", allocationSize = 1)
     @ApiModelProperty(value = "Implementation specific ID for the token in this web service", position = 0, readOnly = true)
     @Column(columnDefinition = "bigint default nextval('token_id_seq')")
+    @JsonView(TokenViews.User.class)
     private long id;
 
     @Column(nullable = false)
     @Convert(converter = TokenTypeConverter.class)
     @ApiModelProperty(value = "Source website for this token", position = 1, dataType = "string")
+    @JsonView(TokenViews.User.class)
     private TokenType tokenSource;
 
     @Column(nullable = false)
     @ApiModelProperty(value = "Contents of the access token", position = 2)
+    @JsonView(TokenViews.Auth.class)
     private String content;
 
     @Column(nullable = false)
     @ApiModelProperty(value = "When an integrated service is not aware of the username, we store it", position = 3)
+    @JsonView(TokenViews.User.class)
     private String username;
 
     @Column()
-    @ApiModelProperty(value = "The ID of the user on the integrated service.", position = 3)
+    @ApiModelProperty(value = "The ID of the user on the integrated service.", position = 7)
+    @JsonView(TokenViews.User.class)
```

Comment:
This was like this, but reviewing got me thinking... Should we be exposing this? @NatalieEO , are the GitHub and Google ids publicly available?

Reply:
GitHub is definitely public, but doesn't seem like it is for Google.

Reply:
Is it okay to expose `onlineProfileId` then? The UI is still using the GitHub token to get the GitHub user id in order to create a GitHub Apps installation link. If the `onlineProfileId` is exposed, then it won't need many changes in the UI since it can get the GitHub user id from this field instead. If it's not exposed, we may need to make an endpoint that retrieves the github user id so the UI doesn't need to use the token to do it (in which case, that needs to be done first before merging this PR).

Reply:
Since I'm somewhat struggling with my Google ticket to gather the `onlineprofileid`s, I can't say definitively that it's private, but it definitely seems that way. Looks like everything requires some sort of authorization for Google, so it's probably better to not expose it. If there's a way to only expose the `onlineprofileid` if it's for GitHub, then that should be okay because anyone can see that by looking at ` https://api.github.com/users/{username}`. If there isn't a way, then the endpoint makes more sense.

Reply:
Since it is like this already, thinking we deal with token content first, ids later?
The route to attack a Google account is much more involved anyway

Reply:
Yeah only the person logged in can see their own id's too which definitely isn't as sensitive as seeing the token anyway.

Reply:
Okay, will keep `onlineProfileId` for this PR, and use it to get the GitHub user id in the UI